### DISCUSSION TITLE
Is it possible to change the log format of the dagster_daemon process?

### DISCUSSION THREAD

**Discussion by NakulK48** [dagster-io/dagster#4150] [Q&A]
Our internal logs are in UTC and have a specific format; this format is not matched by the dagster_daemon. This means that the log file is inconsistent in terms of the format of its messages (in particular the date format) and this makes splunk ingestion tricky.

Is there any way to modify the daemon to log in our desired format?

**Comment by alangenfeld** [comment]
Not currently - we can track its addition in your feature request issue here https://github.com/dagster-io/dagster/issues/4170
